README.md

```diff
@@ -136,6 +136,41 @@ re-executed in order to pick up a non-hermetic change to your environment (e.g.,
 updating your system `python` interpreter), you can completely flush out your
 repo cache with `bazel clean --expunge`.
 
+### Importing `pip` dependencies incrementally (experimental)
+
+One pain point with `pip_install` is the need to download all dependencies resolved by
+your requirements.txt before the bazel analysis phase can start. For large python monorepos
+this can take a long time, especially on slow connections.
+
+To download only the pip packages needed to build targets in the
+subgraph of top level targets in your bazel invocation, you can experiment with using `pip_install_incremental`.
+The interface of `pip_install_incremental` mirrors `pip_install` as closely as possible.
+
+The only user facing difference between `pip_install` and `pip_install_incremental` is that for the latter you need
```

Comment:
not really the only difference, since you also need an extra load() and execute the install_deps function